runtime/runtime-params-estimator/src/main.rs

```diff
@@ -176,6 +196,7 @@ fn main_docker(state_dump_path: &Path) -> anyhow::Result<()> {
 
         let mut buf = String::new();
         buf.push_str("set -ex;\n");
+        buf.push_str("export CARGO_PROFILE_RELEASE_LTO CARGO_PROFILE_RELEASE_CODEGEN_UNITS;\n");
```

Comment:
Do we actually need this `export`? I'd think that it works without it as well, could you check this?

Reply:
Checked, that's correct.